test/core/integration/concrete/lockup/get-refunded-amount/getRefundedAmount.tree

```diff
@@ -1,18 +1,17 @@
-getRefundedAmount.t.sol
-├── given the ID references a null stream
+GetRefundedAmount_Integration_Concrete_Test
+├── given null
 │  └── it should revert
-└── given the ID does not reference a null stream
-   ├── given the stream has been canceled
-   │  ├── given the stream's status is "CANCELED"
-   │  │  └── it should return the correct refunded amount
-   │  └── given the stream's status is "DEPLETED"
-   │     └── it should return the correct refunded amount
-   └── given the stream has not been canceled
-      ├── given the stream's status is "PENDING"
+└── given not null
+   ├── given canceled stream and CANCELED status
```

Comment:
Good point. 

```tree
GetRefundedAmount_Integration_Concrete_Test
├── given null
│  └── it should revert
└── given not null
   ├── given canceled stream
   │  ├── given CANCELED status
   │  │  └── it should return the correct refunded amount
   │  └── given not CANCELED status
   │     └── it should return the correct refunded amount
   └── given not canceled stream
      ├── given PENDING status
      │  └── it should return zero
      ├── given SETTLED status
      │  └── it should return zero
      ├── given DEPLETED status
      │  └── it should return zero
      └── given STREAMING status
         └── it should return zero
```

Is that what you mean then? Because we cannot have two branches that says `given DEPLETED status`.

Reply:
not really, more what i was referring below